api/desecapi/models/tokens.py

```diff
@@ -146,6 +148,16 @@ class TokenDomainPolicy(ExportModelOperationsMixin("TokenDomainPolicy"), models.
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
     token = models.ForeignKey(Token, on_delete=models.CASCADE)
     domain = models.ForeignKey("Domain", on_delete=models.CASCADE, null=True)
+    # TODO use RRset.subname_validators etc?
+    subname = models.CharField(
+        max_length=178,
+        blank=True,
+        null=True,
+        validators=RRset.subname.field._validators,
+    )
+    type = models.CharField(
+        max_length=10, null=True, validators=RRset.type.field._validators
```

Comment:
make sure that effective validators are equal for `TokenDomainPolicy` and `RRset` `subname` and `type`, e.g. by using Django console.

Reply:
done